### DISCUSSION TITLE
Manual argTypes on MDX in Storybook for HTML?

### DISCUSSION THREAD

**Discussion by ghost** [storybookjs/storybook#12867] [Help] [answered]
The ArgsTable in Storybook for HTML seems to only show up when I use CSF, but not with MDX. Am I doing something wrong?

### With MDX:

```jsx
import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks';

export const Template = ({ variant }) => {
  return `
    <div>${variant}</div>
  `;
};

<Meta
  title="Test"
  argTypes={{
    variant: {
      control: {
        type: 'inline-radio',
        options: ['primary'],
      },
    },
  }}
/>

# Test

<Canvas>
  <Story name="test" args={{ variant: 'primary' }}>
    {Template.bind({})}
  </Story>
</Canvas>

## ArgsTable

<ArgsTable />

```

![Screen Shot 2020-10-22 at 12 35 31](https://user-images.githubusercontent.com/40529625/96909062-1b767d00-1463-11eb-8b9b-5b0b99df725c.png)

### With CSF:

```jsx
export default {
  title: 'Test CSF',
  argTypes: {
    variant: {
      control: {
        type: 'inline-radio',
        options: ['primary'],
      },
    },
  },
};

const Template = ({ variant }) => {
  return `
    <div>${variant}</div>
  `;
};

export const Test = Template.bind({});
Test.args = {
  variant: 'primary',
};
```

![Screen Shot 2020-10-22 at 12 37 45](https://user-images.githubusercontent.com/40529625/96909346-86c04f00-1463-11eb-8e3d-2023208f5bc5.png)

**Comment by ghost** [comment, marked as answer]
Nevermind, found the issue. I wasn't setting the story prop in the ArgsTable, `<ArgsTable story="test" />`.

**Reply by yairEO** [reply]
I want the `<ArgsTable>` to affect all the stories in my MDX at the same time, and not per-story like what you did.

I am unable with `<ArgsTable of={...}/>`, tried both with the Component name and the `title` defined in the `<Meta>`
